src/functions/template-tags/post.php

```diff
@@ -63,3 +63,32 @@ function tribe_the_content( $more_link_text = null, $strip_teaser = false, $post
 	echo tribe_get_the_content( $more_link_text, $strip_teaser, $post_id );
 }
 
+/**
```

Comment:
:no_entry_sign: Doc comment for parameter "$post" missing

<sub>hash: ae53a5b87e7ff7b13155eb02f4b09e12</sub>